packages/common/src/utils/analytics/index.ts

```diff
@@ -113,7 +113,18 @@ export function trackWithCooldown(
   track(event, data);
 }
 
-export default function track(eventName, secondArg: Object = {}) {
+export function trackImprovedDashboardEvent(
+  eventName: string,
+  extraInfo: Object = {}
```

Comment:
The `extraInfo` doesn't seem to be used but we can keep it in case.

Reply:
Indeed it's not used, but I wanted to keep it as a way to extend the `track` API.